components/chat_widget/src/components/ocs-chat/ocs-chat.tsx

```diff
@@ -246,6 +246,30 @@ export class OcsChat {
     window.removeEventListener('resize', this.handleWindowResize);
   }
 
+  private addErrorMessage(errorText: string): void {
+    const errorMessage: ChatMessage = {
+      created_at: new Date().toISOString(),
+      role: 'system',
+      content: `**Error:** ${errorText}\nPlease try again or contact support if the problem persists.`,
```

Comment:
nit: unclear who 'support' is or how to contact them so let's leave it out for now.
```suggestion
      content: `**Error:** ${errorText}\nPlease try again.`,
```